### ISSUE TITLE
1.23.15-Patch change

### ISSUE THREAD

**Issue by hm2075** [muhammadsammy/free-omnisharp-vscode#28]
Please upload a new version to openvsx,     gitpod thinks 1.23.15-patch is older than 1.23.15 and automatically installs the non patch version

i managed to build from source and got the extension plus omnisharp working without issue but new workspaces just download extensions again.  

If possibly can you allow 1.57 vs code?    self hosted gitpod  currently runs on 1.57  unless using nightly build

**Comment by muhammadsammy** [comment]
I don't want to pump up the version in order to match versions with upstream. Try to disable automatic extension updates in gitpod or vscodium by adding `"extensions.autoUpdate": false` to the settings.json until a next upstream version released.

**Comment by hm2075** [comment]
fair enough :)

thanks

**Comment by hm2075** [comment]
I don't think that will work because gitpod downloads extensions each time from open vsx for new workspaces.    So you always end up with the latest which in this is always 1.23.15  and not patch

Its not really your extension issue but maybe replace 1.23.15 altogether?

**Comment by muhammadsammy** [comment]
Unfortunately OpenVSX does not allow removing published versions or replacing them
If gitpod allows adding extensions from source, this build artifact should work https://github.com/muhammadsammy/free-omnisharp-vscode/suites/3800587607/artifacts/93838888 it has the extension with correct debugger links and versioned as `1.23.15`

**Comment by hm2075** [comment]
yes,    can build from source and its fine  but then each user has to manually install the extension for each new workspace on gitpod

**Comment by muhammadsammy** [comment]
I can't think of any solution to this except pumping up the version to 1.23.16 but this will not be matching with upstream. 

Since the official omnisharp team just [published](https://github.com/OmniSharp/omnisharp-vscode/releases/tag/v1.23.16) a prelease of 1.23.16, this issue will be fixed soon

https://github.com/muhammadsammy/free-omnisharp-vscode/commit/76e27ea007b3cca8067a39774d5f0908f19182d0 NetCoreDbg versions will be fixed and updated with each release of this extension in order to avoid this issue

**Comment by hm2075** [comment]
Download the patch version and install it manually

Or  download the source and compile it yourself

**Comment by muhammadsammy** [comment]
`v1.23.16` was released on OpenVSX with working debugger links :+1:
